patch/0001-armhf-mandatory-configs-for-boot.patch

```diff
@@ -0,0 +1,201 @@
+From a961464da6c88aee4d94af4038c748a1e8a1f412 Mon Sep 17 00:00:00 2001
+From: Antony Rheneus <arheneus@marvell.com>
+Date: Thu, 17 Oct 2019 18:57:48 +0530
+Subject: [PATCH] armhf mandatory configs for boot
+
+---
+ debian/build/build_armhf_none_armmp/.config | 53 ++++++++++++++++-------------
+ 1 file changed, 30 insertions(+), 23 deletions(-)
+
+diff --git a/debian/build/build_armhf_none_armmp/.config b/debian/build/build_armhf_none_armmp/.config
+index e997934..69288eb 100644
+--- a/debian/build/build_armhf_none_armmp/.config
++++ b/debian/build/build_armhf_none_armmp/.config
+@@ -130,14 +130,14 @@ CONFIG_CGROUPS=y
+ CONFIG_PAGE_COUNTER=y
+ CONFIG_MEMCG=y
+ CONFIG_MEMCG_SWAP=y
+-# CONFIG_MEMCG_SWAP_ENABLED is not set
++CONFIG_MEMCG_SWAP_ENABLED=y
```

Comment:
>CONFIG_MEMCG_SWAP_ENABLED [](start = 1, length = 25)

why do you need it? we do not use swap at all.

Reply:
I will check this and update if not needed.

Reply:
Done, removed it